frontend/src/js/controllers/authCtrl.js

```diff
@@ -115,9 +115,10 @@
                                 }
 
                             } catch (error) {
-                                console.log(error);
```

Comment:
Please replace `Your data is invalid.` with `Some error occurred. Please try again later. `

Reply:
@deshraj I think we can replace it with the same `error` as in the `console.log` statement. What do you say ?

Reply:
Ah agree. Thanks for the correction. @Marlysson can you please show the `error` variable instead of the text that I posted above?

Reply:
Ok. Just a minute.